### DISCUSSION TITLE
NPM Audit issue. No releases since 2.0.2 two years ago.

### DISCUSSION THREAD

**Discussion by kgoodridge** [silentmatt/expr-eval#272] [Q&A]
Hey Matt,

   I see there was a PR merged to master https://github.com/silentmatt/expr-eval/pull/252 for a security fix. It's showing up as a problem in npm audit. There hasn't been a tag or release in two years.

   Can it be tagged and released?

Thanks,
Ken

**Comment by ds-albans** [comment]
@silentmatt help please.
